### ISSUE TITLE
Continue game on another device

### ISSUE THREAD

**Issue by larry801** [freeboardgames/FreeBoardGames.org#702]
**Is your feature request related to a problem? Please describe.**
Player cannot continue current game on another device

**Describe the solution you'd like**
Add a feature to export jwt token or a url with token to transfer between different clients.

**Describe alternatives you've considered**
Implement an account system login/ register etc.

**Comment by cwatsonc** [comment]
like this idea, but think portable jwts aren't available (or have good reasons not to be).  I did a few years in Identity and believe you're describing a form of federating users, where a seamless experience exists between many screens. This could be done with public IDP's and a limited amount of effort.

**Comment by flamecoals** [comment]
I agree this is a good idea! However, i don't think that sharing the JWT is the way to go, as the backend wouldn't be able to know about each device.

I think the better solution is allow for players either to optionally sign up/login, and we could tie several JWTs to the same account. This would also help with the idea of competitive gaming/having an ELO score for each game
